cmd/controller/main.go

```diff
@@ -433,6 +521,22 @@ type httpServer struct {
 	http.ServeMux
 }
 
+func buildConfig(kubeconfig string) (*rest.Config, error) {
```

Comment:
figure out if we need this? we presumably construct a client elsewhere and that would need similar logic?